### ISSUE TITLE
Icon: #icon-warehouse and #icon-manufacturing are identical in uplift

### ISSUE THREAD

**Issue by jonathanamiran** [infor-design/enterprise#5934]
**Describe the bug**
Warehouse icon in Uplift (New) is different than Classic but identical to manufacturing.

**To Reproduce**

1. Go to [https://main-enterprise.demo.design.infor.com/components/icons/example-index.html?theme=new&mode=light](https://main-enterprise.demo.design.infor.com/components/icons/example-index.html?theme=new&mode=light)
2. See icon-warehouse

**Expected behavior**
Warehouse icon should be redesigned.

**Version**

- ids-enterprise: v4.58.0

**Screenshots**
![image](https://user-images.githubusercontent.com/34129313/145398273-2f9b9492-9b9c-4be4-9945-b4b108ee6b9c.png)

**Platform**

 - OS Version: Windows 10
 - Browser Name: Chrome
 - Browser Version: 96 (Official Build) (64-bit)]

**Additional context**
Thanks!

**Comment by tmcconechy** [comment]
I believe the design team did this on purpose in order to reduce the number of different icons and remove ones that are a bit irregular. So checking with @kevinwhitedesign

**Comment by tmcconechy** [comment]
cc @elizabethhartley

**Comment by kevinwhitedesign** [comment]
Yes I believe it was done to reduce the amount of icons we support.
